src/main/java/weavers/siltarae/global/exception/ExceptionCode.java

```diff
@@ -21,6 +21,7 @@ public enum ExceptionCode {
     NOT_SUPPORTED_AUTH_SERVICE(5001, "지원하는 소셜 로그인 서비스가 아닙니다."),
     INVALID_AUTHORIZATION_CODE(5002, "인증 코드가 유효하지 않습니다."),
     AVAILABLE_AFTER_LOGIN(5003, "로그인 후 이용 가능합니다."),
+    MEMBER_EMAIL_ISNULL(5004, "이메일은 빈값일 수 없습니다."),
```

Comment:
```suggestion
    INVALID_EMAIL(4002, "유효하지 않은 이메일입니다.")
```

이건 어떨까요...? 👀 4000번대를 제안드리는 이유는
- 이메일 필드는 Member 도메인에 속해있는 상태
- 추후 이메일 수정 기능이 생기면, 이메일이 유효한 형식인지도 (@가 들어갔는지 등등) 체크해야 하지 않을까요?

물론 지금 상태로도 괜찮습니다 🙌 편한대로 해주시면 될 것 같아요!